src/theme/common.scss

```diff
@@ -620,6 +620,12 @@ strong {
   background-color: mat.get-color-from-palette($sormas-warning, 800);
 }
 
+.case-person-contact-form {
```

Comment:
please remove this from here and apply the code in the desired component

Reply:
Done.